src/test/java/com/softserveinc/ita/vsaroz/SignUpTest.java

```diff
@@ -0,0 +1,45 @@
+package com.softserveinc.ita.vsaroz;
+
+import com.codeborne.selenide.Configuration;
+import org.testng.annotations.*;
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selenide.$x;
+import static com.codeborne.selenide.Selenide.open;
+
+public class SignUpTest {
+
+    private String firstName = "UsersFirstName";
```

Comment:
create model of User, and Repo class, and init user object in repo and get it from there